net_util/src/tap.rs

```diff
@@ -40,7 +40,7 @@ pub type Result<T> = ::std::result::Result<T, Error>;
 #[derive(Debug)]
 pub struct Tap {
     tap_file: File,
-    if_name: [u8; 16usize],
+    if_name: Vec<u8>,
```

Comment:
If if_name is defined as [u8; 16usize], it means it has a fixed length, as a result, it will always fail when the tap is opened the second time if we provide a shorter name (< 16) from the command line for the first opening. Change it to a vector and update the associated logic in open_named, to ensure it will have the same name to open for many times.